docs/pages/api.rst

```diff
@@ -723,6 +723,27 @@ Signature: ``Collection::duplicate(?callable $comparatorCallback = null, ?callab
 .. literalinclude:: code/operations/duplicate.php
   :language: php
 
+entropy
+~~~~~~~
+
+This method extends the library's functionality by allowing users to calculate
+the normalized Shannon entropy of a collection.
+
+The implementation provides the normalized version of Shannon entropy.
+Normalization ensures that the entropy value is scaled between `0` and `1`,
+making it easier to compare entropy across different collections, irrespective
+of their size or the diversity of elements they contain.
+
+An entropy of `0` indicates no diversity (all elements are the same), while an
+entropy of `1` signifies maximum diversity.
```

Comment:
would be useful to clarify what the resulting collection means, because just by reading this I would assume that the operation would return a single value. But actually it results in a list of values.